### PR TITLE
Installing kernel extensions using scripts

### PR DESCRIPTION
**Pull request by gniezen**
This PR for the wiki describes how to install macOS kernel extensions using a script.

### CODE REVIEW

File: docs/Kernel extensions.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+Installing macOS kernel extensions with `electron-builder` can be done using scripts.
+
+First, make sure you're building a package:
+
+```json
+"mac": {
+	{
+		"target": "pkg",
+		"arch": [ "x64" ]
```
Comment (develar) [inline docs/Kernel extensions.md]: Not required. Please remove to make config more smaller.

File: docs/Kernel extensions.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	}
+}
+```
+
+Place your script and the kernel extensions in `resources/pkg-scripts`, or [define a custom directory](https://github.com/electron-userland/electron-builder/wiki/Options#PkgOptions-scripts). Note that the script **must** be called either `preinstall` or `postinstall`.  Remember to use ` #!/bin/sh` as the first line in your script. Also, your script must be executable (`chmod +x <filename>`).
```
Comment (develar) [inline docs/Kernel extensions.md]: > `resources/pkg-scripts`

By default `build/pkg-scripts`

**Comment by develar** [comment]
So cool. Thanks a lot for tutorial. Please fix mentioned small issues and will be merged. Later I will add links to this tutorial from various places.

**Comment by develar** [comment]
Thanks a lot. Wiki will be synced tomorrow morning CET.

**Comment by gniezen** [comment]
No problem. Thanks for the quick review and merge! 😄

**Pull request merged**